### ISSUE TITLE
FEATURE REQUEST: Add Just Player to Built-in Media Playback Options

### ISSUE THREAD

**Issue by pmindemann11** [Stremio/stremio-features#1023]
After battling through several days of being unable to get any material to have properly syncronized audio through my Nvidia Shield (even trying all of the usual hardware and software "fixes"), I finally tried to use an external player. So, I loaded both MX Player and Just Player onto my Shield, and viola! BOTH of them fixed all of the audio/video sync issues!

From what I've read, Just Player is basically an updated version of Exo Player, so it seems like it would "play nice" with the existing Stremio framework. MX Player worked, too, but seemed a lot more complicated than necessary and didn't seem as intuitive. 

So, I've come here to request that Just Player be added as a built-in media player within the standard Stremio build. As many people have said... it JUST works!

**Comment by TRtomasz** [comment]
just player isnt library, media3 is library and we use media3 inside stremio
once media3 fix bugs that are preventing us from updating it in stremio we will update media3 to the latest version too

**Comment by pmindemann11** [comment]
Thanks for the reply. Unfortunately, I'm not familiar enough with the inner workings to understand what that means (is or isn't "library"), but I appreciate the explanation. All I know is that I spent days trying to figure out why audio wasn't sync'ing with the video, and nothing I tried made any difference until I used an external player. I'm glad I didn't give up on Streamio before I figured that out!
